builds/jazz-build-module/service-metadata-loader.groovy

```diff
@@ -55,7 +55,13 @@ def loadServiceMetadata(service_id){
 			metadata['created_by'] = service_data.Item.SERVICE_CREATED_BY.S
 			metadata['type'] = service_data.Item.SERVICE_TYPE.S
 			metadata['runtime'] = service_data.Item.SERVICE_RUNTIME.S
-			metadata['region'] = configLoader.AWS.REGION
+			if(service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS){
+				metadata['accountId'] = service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS.L[0].M.accountId.S
+        metadata['region'] = service_data.Item.SERVICE_DEPLOYMENT_ACCOUNTS.L[0].M.region.S
```

Comment:
indentation here and other places

Reply:
done

Reply:
indentation seems to be an issue. i think each one of us are using different tab settings. better yet is to just use spaces instead of tabs and set each tab to 4 characters. you can view the spaces (if they are spaces or tabs) in Visual Code using the following menu:
![image](https://user-images.githubusercontent.com/1162453/58211920-d9b43680-7cb2-11e9-827a-e8040aef08bc.png)